content/python/concepts/os-path-module/terms/join/join.md

```diff
@@ -0,0 +1,37 @@
+---
+Title: '.join()'
+Description: 'Returns a string of pathnames delimited with a forward-slash.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Libraries'
+  - 'Modules'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.join()`** method returns a string of pathnames delimited with a forward-slash.
+
+## Syntax
+
+```pseudo
+os.path.join(path_1, path_2,..., path_n)
+```
+
+Before being added to the returned value, each subsequent `path_` string is prepended with a forward-slash `/`.
+
+## Codebyte Example
+
+The following example showcases the `.join()` method:
+
+```codebyte/python
```

Comment:
It may be helpful to make this a functional url instead of just an abstract example